app/templates/_service_meta.html

```diff
@@ -3,7 +3,11 @@ <h2 class="visuallyhidden">Pricing</h2>
   <p class="price">{{ service.meta.price }}</p>
   <ul class="price-caveats">
   {% for caveat in service.meta.priceCaveats %}
-    <li>{{ caveat }}</li>
+    <li>
```

Comment:
There could be an argument for using the toolkit external link style rather than baking the HTML ourselves here...
```
{%
  with
  text = caveat['text'],
  link = caveat['link'],
  target = "_blank"
%}
  {% include "toolkit/external-link.html" %}
{% endwith %}
```